### DISCUSSION TITLE
Welcome to DSC DYPCOE Community!

### DISCUSSION THREAD

**Discussion by Akash1362000** [DSC-DYPCOE/Learning-Resources#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.
Use this discussion forum for project ideas, team formation, collaboration & networking.

### Feel free to use this discussion forum for [Google DSC Solution Challenge](http://goo.gle/solutionchallenge) discussions as well
---
#### Resources that might help you get started with Solution Challenge:
* [Slide Deck](http://bit.ly/solution-challenge-resources)
* [Official Website](http://goo.gle/solutionchallenge)
* [Introduction Session on DSC Solution Challenge](https://www.youtube.com/watch?v=tJO7ibU4VnA)

**Comment by harshkumar02** [comment]
Hi there,
I am Harsh
A first year CSE student.
I am looking to join a team for Google solution challenge.
I have good knowledge of Django, Kotlin and Python.
